test/jdk/java/nio/file/Files/StreamTest.java

```diff
@@ -75,7 +75,7 @@ public class StreamTest {
     static boolean supportsLinks;
     static Path[] level1;
     static Path[] all;
-    static Path[] all_folowLinks;
+    static Path[] all_followLinks;
```

Comment:
The name `all_followLinks`, which contains the underline and the lower camel case, is strange. What about `allFollowLinks`?